README.md

```diff
@@ -1 +1,52 @@
-# first-review
\ No newline at end of file
+# The Next Web Clone
+
+> This is a clone of The Next Week page.
+
+This is the header section
+![screenshot](./img/header.png)
+This is the latest news section
+![screenshot](./img/latest.png)
+This is the footer section
+![screenshot](./img/footer.png)
+
+
+In this project, I am working on making the website responsive.
+
+## Built With
+
+- Html
+- Css
+
```

Comment:
This is a good README file. If you look at https://github.com/microverseinc/readme-template you will see that there is a setup section. Adding this to your project will help any non-technical user to set up your work on a local machine.
Example: 
- git clone your repository
- copy to a folder and open with live server
Kindly update this

Reply:
Good job using inline comments here!

**Optional**
- [ ] A nice tip to consolidate your reviews under a single summary is to click the "Start a review" button when adding the first review comment. This helps to house all your inline comments under one single summary like this current review. :wink:
This is an example inline comment. :slightly_smiling_face:

You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. :wink:
To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.